### PR TITLE
VTCCA-4617 loose ends

### PR DESCRIPTION
**Pull request by darrell-plant**
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4617

### CODE REVIEW

File: app/views/propertyLinking/relationshipToProperty.scala.html [hunk 1]
```diff
@@ -90,21 +94,16 @@ <h1 class="govuk-heading-l">@messages("declarePropertyRelationship.title"):</h1>
...
             name = "capacity",
             items = CapacityType.options.map { item =>
                 RadioItem(
                     value = Some(item),
-                    content = Text(messages(s"capacity.$item")),
-                    checked = form.data.get("capacity").fold(false)(_ == item)
+                    content = Text(messages(s"capacity.$item"))
                 )
             },
-            errorMessage =  getFiledErrorMessage("capacity"))
```
Comment (reddy-yattapu) [inline app/views/propertyLinking/relationshipToProperty.scala.html:102]: errorMessage =  getFiledErrorMessage("capacity") might not required as you have added .withFormField(form("capacity"))

Comment (darrell-plant) [reply app/views/propertyLinking/relationshipToProperty.scala.html:102]: done - and removed other redendant code

**Comment by reddy-yattapu** [comment]
👍

File: app/views/propertyLinking/relationshipToProperty.scala.html (hunk 1 above)
Comment (reddy-yattapu) [reply app/views/propertyLinking/relationshipToProperty.scala.html:102]: thanks mate

**Comment by PeterPerhac** [comment]
👍  looks good to me

**Pull request merged**